### PR TITLE
[DOC-12077] base64 Vector Embeddings (SERVER VERSION)

### PR DESCRIPTION
**Pull request by sarahlwelton**
With 7.6.2, Vector Search will support a new field data type, `vector_base64`, which will let users index and query vectors stored as base64 encoded strings, instead of arrays. 

These changes support the new field data type and query object specific to base64 vectors - for the Server documentation.

### CODE REVIEW

File: modules/vector-search/pages/create-vector-search-index-ui.adoc [hunk 1]
```diff
@@ -61,22 +62,25 @@ You cannot have 2 indexes with the same name inside the same bucket and scope.
 .. In the *Field* field, enter the name of the field in your documents that contains your vector embeddings.
 +
 Vectors must be represented as an array of floating point numbers. 
-.. In the *Type* list, select *vector*. 
+.. In the *Type* list, select do one of the following:
```
Comment (bfavini) [inline modules/vector-search/pages/create-vector-search-index-ui.adoc]: @sarahlwelton should you delete "select here? "select do one of the following"?

File: modules/vector-search/pages/run-vector-search-ui.adoc [hunk 2]
```diff
@@ -102,6 +102,27 @@ Click btn:[View on GitHub] to view and copy the entire Vector Search query paylo
...
+----
+
+The following query uses a base64 encoded string for the same query as <<large,Running a Semantic Search Query with a Large Embedding Vector>> to return the document for `navy`: 
+
+[source,json]
+----
+include::example$run-vector-search-payload-base64-ui.jsonc[]
+----
+
+NOTE: You can only use base64 encoded strings in your Vector Search queries if your documents use base64 encoded strings, indexed with the *vector_base64* field data type.
```
Comment (bfavini) [inline modules/vector-search/pages/run-vector-search-ui.adoc]: You can use base64 encoded strings in your Vector Search only if your documents use base64-encoded strings...

**Review by bfavini — APPROVED** [review]
couple of small comments

File: modules/vector-search/pages/create-vector-search-index-ui.adoc (hunk 1 above)
Comment (sarahlwelton) [reply modules/vector-search/pages/create-vector-search-index-ui.adoc]: See this is why I wanted a review :D Thank you!

**Pull request merged**